README.md

```diff
@@ -1,4 +1,11 @@
-# Splunk Distribution of OpenTelemetry .NET
+# Splunk Distribution of OpenTelemetry Go
+
+[![OpenTelemetry Dotnet](https://img.shields.io/badge/OTel-1.3.1-blueviolet)](https://github.com/open-telemetry/opentelemetry-dotnet/releases/tag/core-1.3.1)
```

Comment:
I think that we should have a link to AutoInstrumentation instead of the OTel .NET (or both of them).